src/helpers/testing/mockLogger.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Import this file from test scripts to mock the default logger.
+ * To simply overwrite the default logger to prevent output, use
+ * `import {} from './testing/mockLogger';`
+ * If you wish to monitor the usage of one of the endpoints, you can import
+ * the specific endpoints you want. For example, if you wish to track error output, use
+ * `import { error as mockErrorLogger } from './testing/mockLogger';`
+ * `expect(mockLoggerWarn).toHaveBeenCalledWith('warn');`
+ */
+
+jest.mock('../../logger');
+import { getLogger } from '../../logger';
+const mockGetLogger = getLogger as jest.Mock;
+
+const mockLogger = {
+	debug: jest.fn(),
+	info: jest.fn(),
+	warn: jest.fn(),
+	error: jest.fn(),
```

Comment:
Since you reference these down below in a way that TS thinks is referencing unbound methods, it might be best to declare and export these separately, and then construct `mockLogger`.